src/runner.py

```diff
@@ -151,3 +177,18 @@ def generate_plots(self, scale_folder: str, susc: float, base_r0: float):
             filename="Grads_tri.pdf",
             folder=scale_folder
         )
+
+        # instantiate contact manipulation and get the plot
+        contact_manipulation = ContactManipulation(
```

Comment:
Why don't you pass here only the `self.sens_calc`, `susc` and `base_r0`? The constructor of `ContactManipulation` could get everything else from `self.sens_calc`.